test/unit/resources/parse_config_test.rb

```diff
@@ -24,4 +24,13 @@
     _(resource.params).must_equal result
     _(resource.send("kernel.domainname")).must_equal "example.com"
   end
+
+  it "parse_config resource accepts arrays" do
```

Comment:
Maybe we should mention the rspec `its` case here. In future, we might read this test and wonder why we needed to support this case.

Reply:
I modified it to say `parse_config resource accepts arrays due to rspec's its behavior`, does that look good enough to you?

Reply:
Brilliant, thank you!